### DISCUSSION TITLE
想麻烦大佬做一个soul app的规则

### DISCUSSION THREAD

**Discussion by zaozaodounai** [blackmatrix7/ios_rule_script#1014] [Q&A]
这几天一直在研究切换 soul 归属地的事情，iphone 虚拟定位+vpn 全局代理模式亲测可用，但是群聊功能会受限，所以想通过关闭 app 定位+vpn 配置规则的方式来进行伪造。 我开始尝试抓包，发现 soul 真的丧心病狂，居然找了好几家专门做校验和设备识别的公司提供服务 fengkongcloud ，shuzilm ，尝试将抓包的所有域名和 ip 都配置到规则里了，结果还是不能正确切换 ip 归属地，想麻烦大佬研究一下

**Comment by heichaowo** [comment]
也许这个规则可以帮到你
https://github.com/rule/app/SOUL/SOUL.list

**Reply by simidabinbin** [reply]
有这个规则怎么操作呢

**Comment by zaozaodounai** [comment]
> 也许这个规则可以帮助你 https://github.com/rule/app/SOUL/SOUL.list

感谢，我在您的仓库里找到了脚本，但是我用的是shadowrocket，能再次麻烦您指导下吗

**Reply by heichaowo** [reply]
> shadowrocket

直接添加规则rule-set填入规则的raw地址就OK了

**Comment by mary07058888-crypto** [comment]
大佬，我最近也在研究这个问题，不知道您现在发现解决方案了没有，只不过我的手机是安卓的，试用了虚拟位置只能骗soul一分钟，如果您有解决方案的话能否提供给我，可有偿
